services/setup/public/index.html

```diff
@@ -1,26 +1,63 @@
 <!DOCTYPE html>
 <html>
-
 <head>
   <title>Radiodan Configuration</title>
   <style>
+    .hidden { display: none; }
     button, label, span { display: block }
     button { margin: 10px 0; width: 50% }
     span { float: left; width: 50%; }
   </style>
 </head>
-
 <body>
 <h1>Radiodan Configuration</h1>
+<p id="notification" class="hidden">
+  Changes have been saved, please reboot your Pi for them to take effect.
+</p>
+
+<h2>Application</h2>
+<form action="/apps" method="POST" id="apps-list">
+  <label>
+    <span>All Apps</span>
+    <input type="radio" name="apps" value="" checked="checked" />
+  </label>
+  <button type="submit">Update</button>
+</form>
 
 <h2>Services</h2>
 <form action="/services" method="POST" id="services-list">
   <button type="submit">Update</button>
 </form>
 
 <script>
-  const app = async () => {
+  const buildApps = async () => {
+    const form = document.getElementById('apps-list');
+    const selection = document.createElement('div');
+    const apps = await fetch('/apps').then(res => res.json());
+
+    apps.forEach(({ name, active }) => {
+      const label = document.createElement('label');
+      const span = document.createElement('span');
+      span.innerText = name;
+
+      label.appendChild(span);
+
+      const radio = document.createElement('input');
+      radio.type = 'radio';
```

Comment:
Does this mean only one app is enabled at any time? If so, I think we should leave this up to the person configuring the device, rather than apply the limitation here.

Reply:
The way I made the changes to https://github.com/andrewn/neue-radio/pull/27 enforces this limitation. Can you think of a situation where you'd want to run a specific sub-set of applications?

We already have scenarios covered for:
* A single application
* All installed applications

Reply:
Since switching apps requires a reboot,  I thought that perhaps you'd want to run a couple of apps at the same time. Perhaps the `radio` app to test sound output and another app to demo another set of features?

They're not very specific examples and if it's a big change then perhaps we should leave it until we have a concrete use-case. But it just seems a bit strange to me to enforce running one app or all of them.

Reply:
I'll see if I can set an array as an environment variable